doc/python/facet-plots.md

```diff
@@ -118,3 +118,20 @@ fig.show()
 ```python
 
 ```
+
+### Zooming subplots to the same range
```

Comment:
`matches` does actually more than synchronizing zoom, you also have synchronized panning for example. How about a more general title like "Synchronizing axes in subplots with `matches`"? Also what do you think of mentioning pan and zoom in the text below (compared to just zoom at the moment?)

Reply:
Sure! I will edit the title and mention the pan.